src/backend/base/langflow/components/tools/file_manipulation.py

```
@@ -0,0 +1,1322 @@
+                    replaced_text = re.sub(search, replacement, match_text)
+                    new_content = new_content[:pos_start] + replaced_text + new_content[pos_end:]
+                except re.error:
+                    # Skip invalid regex replacements
+                    continue
+            else:
+                new_content = new_content[:pos_start] + replacement + new_content[pos_end:]
+
+        return new_content, to_replace
+
+    def build_toolkit(self) -> Tool:
+        """Creates and returns a list of tools for file and directory operations within the workspace.
```

Pylint (3.3.7)

[convention] Line too long (102/100)

(C0301)